app-backup/amanda/amanda-3.5.1-r3.ebuild

```diff
@@ -403,42 +426,41 @@ src_install() {
 	dodoc AUTHORS ChangeLog DEVELOPING NEWS README ReleaseNotes UPGRADING
 	# our inetd sample
 	einfo "Installing standard inetd sample"
-	newdoc "${MYFILESDIR}/amanda-inetd.amanda.sample-2.6.0_p2-r2" amanda-inetd.amanda.sample
+	newdoc "${MYFILESDIR}"/amanda-inetd.amanda.sample-2.6.0_p2-r2 amanda-inetd.amanda.sample
 	# Amanda example configs
 	einfo "Installing example configurations"
-	rm "${D}"/usr/share/amanda/{COPYRIGHT,ChangeLog,NEWS,ReleaseNotes} || die
-	mv "${D}/usr/share/amanda/example" "${D}/usr/share/doc/${PF}/" || die
+	rm "${ED}"/usr/share/amanda/{COPYRIGHT,ChangeLog,NEWS,ReleaseNotes} || die
+	mv "${ED}"/usr/share/amanda/example "${ED}"/usr/share/doc/${PF}/ || die
 	docinto example1
 	newdoc "${FILESDIR}"/example_amanda.conf amanda.conf
 	newdoc "${FILESDIR}"/example_disklist-2.5.1_p3-r1 disklist
 	newdoc "${FILESDIR}"/example_global.conf global.conf
 
 	einfo "Cleaning up dud .la files"
 	perl_set_version
-	find "${D}"/"${VENDOR_LIB}" -name '*.la' -delete || die
+
+	find "${ED}" -name '*.la' -delete || die
 }
 
 pkg_postinst() {
-	[ ! -f "${TMPENVFILE}" -a "$MERGE_TYPE" == "binary" ] && \
-		TMPENVFILE="${ROOT}${ENVDIR}/${ENVDFILE}"
-	[ ! -f "${TMPENVFILE}" ] && die "Variable setting file (${TMPENVFILE}) should exist!"
+	[[ ! -f ${TMPENVFILE} && ${MERGE_TYPE} == "binary" ]] && TMPENVFILE="${EROOT}${ENVDIR}/${ENVDFILE}"
+	[[ ! -f ${TMPENVFILE} ]] && die "Variable setting file (${TMPENVFILE}) should exist!"
 	source "${TMPENVFILE}"
 
 	# Migration of amandates from /etc to $localstatedir/amanda
-	if [ -f "${ROOT}/etc/amandates" -a \
-		! -f "${ROOT}/${AMANDA_USER_HOMEDIR}/amanda/amandates" ]; then
+	if [[ -f ${EROOT}/etc/amandates && ! -f ${EROOT}/${AMANDA_USER_HOMEDIR}/amanda/amandates ]]; then
 		einfo "Migrating amandates from /etc/ to ${AMANDA_USER_HOMEDIR}/amanda"
 		einfo "A backup is also placed at /etc/amandates.orig"
-		cp -dp "${ROOT}/etc/amandates" "${ROOT}/etc/amandates.orig" || die
-		mkdir -p "${ROOT}/${AMANDA_USER_HOMEDIR}/amanda/" || die
-		cp -dp "${ROOT}/etc/amandates" "${ROOT}/${AMANDA_USER_HOMEDIR}/amanda/amandates" || die
+		cp -dp "${EROOT}"/etc/amandates "${EROOT}"/etc/amandates.orig || die
+		mkdir -p "${EROOT}/${AMANDA_USER_HOMEDIR}/amanda/" || die
+		cp -dp "${EROOT}"/etc/amandates "${EROOT}"/${AMANDA_USER_HOMEDIR}/amanda/amandates || die
 	fi
-	if [ -f "${ROOT}/etc/amandates" ]; then
+	if [[ -f ${EROOT}/etc/amandates ]]; then
```

Comment:
"${EROOT}"?

Reply:
I don't think `[[ ... ]]` does word splitting

Reply:
correct, which is why I removed the quotes